wp-content/themes/csisjti/template-parts/event-block-upcoming.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * The default template for displaying content
+ *
+ * Used for both singular and index.
+ *
+ * @link https://developer.wordpress.org/themes/basics/template-hierarchy/
+ *
+ * @package CSIS iLab
+ * @subpackage @package CSISJTI
+ * @since 1.0.0
+ */
+
+?>
+	<div class="event-block">
+
+		<?php if( get_field('date_of_event') ): ?>
+		<div class="event-block__date">
+			<?php $date = split_date(get_field('date_of_event')); ?>
+				 <p class="event-block__month"><?php echo $date[1] ?></p>
```

Comment:
I would make these `<div>` instead of paragraphs, since they're not really paragraphs of text.